app/serializers/api/v0_1/trainee_serializer.rb

```diff
@@ -49,6 +49,8 @@ def as_hash
             training_route: training_route,
             nationality: nationality,
             training_initiative: training_initiative,
+            withdrawal_trigger: @trainee.current_withdrawal&.trigger,
+            withdrawal_future_interest: @trainee.current_withdrawal&.future_interest,
             withdraw_reasons: withdraw_reasons,
```

Comment:
Is this relying on the old relationship in `models/trainee.rb`?

https://github.com/DFE-Digital/register-trainee-teachers/blob/a5eb30e86f1648211cfaa8a47cc1d892724f1b45/app/models/trainee.rb#L188-L192